recipe/meta.yaml

```
@@ -21,101 +21,18 @@ requirements:
     - pip
   run:
     - python
-    - langchain-core >=0.1.52,<0.2.0
-    - sqlalchemy >=1.4,<3
+    - langchain <0.4.0,>=0.3.11
+    - langchain-core <0.4.0,>=0.3.24
```

https://github.com/langchain-ai/langchain/blob/langchain-community%3D%3D0.3.12/libs/community/pyproject.toml#L34

```suggestion
    - langchain-core <0.4.0,>=0.3.25
```